src/Digdir.Domain.Dialogporten.Application/Features/V1/EndUser/Dialogs/Queries/Search/MappingProfile.cs

```diff
@@ -14,7 +14,6 @@ public MappingProfile()
     {
         CreateMap<DialogEntity, SearchDialogDto>()
             .ForMember(dest => dest.LatestActivity, opt => opt.MapFrom(src => src.Activities
-                .Where(activity => activity.TypeId != DialogActivityType.Values.Forwarded)
```

Comment:
You know whuy we have excluded this in the past @oskogstad ? And if we need to exclude any other activities based on type?

Reply:
The forwarded event was meant to describe user driven delegations on single dialogs. This is not yet supported in altinn authorization, so we should just remove it from the enum (and thus this exclusion check) for now. No other statuses should be excluded either.